src/pages/experiments/[experimentId]/data-exploration/components/differential-expression-tool/DiffExprResults.jsx

```diff
@@ -73,6 +73,9 @@ const DiffExprResults = (props) => {
       ),
     );
   };
+  const optionName = (word) => word.split('/').pop();
+  const { basis, cellSet, compareWith } = comparisonGroup[comparisonType];
+  const settingsText = `${optionName(basis)} vs. ${optionName(cellSet)} in ${optionName(compareWith)}`;
```

Comment:
- This should be `cellSet` vs `compareWith` in `basis`. 
- I think it'd be better to use the "key" name instead of the Uuid. e.g. `Louvain 0`, `Louvain 1` and `Control`.
- Can you make the names (basis, cellSet, compareWith) in bold? so that they're easier to spot

<img width="354" alt="image" src="https://user-images.githubusercontent.com/2862362/106263152-96b3b000-6256-11eb-90d3-85e1780cfe68.png">